module/optee/voltd_regulator/include/mod_optee_voltd_regulator.h

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2023, Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef MOD_OPTEE_VOLTD_REGULATOR_H
+#define MOD_OPTEE_VOLTD_REGULATOR_H
+
+#include <fwk_element.h>
+#include <fwk_macros.h>
+
+#include <stdint.h>
+#include <stdbool.h>
+
+struct scmi_server_regu_channel;
+struct regulator;
+
+/*!
+ * \brief Platform regulator configuration.
+ */
+struct mod_optee_voltd_regulator_dev_config {
+    struct regulator *regulator;
+    bool default_enabled;
+};
+
+/*
+ * Build SCMI server resource tables for the voltage domains exposed
+ * by platform.
+ *
+ * @plat - reference to platform data
+ * @plat_count - Number of elements in platform data
+ */
+void scmi_server_build_optee_regu_config(struct scmi_server_regu_channel *plat,
+                                         size_t plat_count);
+
```

Comment:
isn't possible to expose them as an framework API? or what is the reason to expose the prototype directly?

Reply:
Sorry, this is a leftover from another development. This function is not implemented neither called. I'll remove this line.